Libraries/CustomComponents/Navigator/Navigator.js

```diff
@@ -129,54 +129,168 @@ var GESTURE_ACTIONS = [
 ];
 
 /**
- * Use `Navigator` to transition between different scenes in your app. To
- * accomplish this, provide route objects to the navigator to identify each
- * scene, and also a `renderScene` function that the navigator can use to
- * render the scene for a given route.
+ * `Navigator` handles the transition between different scenes in your app.
+ * You should consider using this component instead of `NavigatorIOS` if you're
+ * building a cross-platform app. `Navigator` is implemented in JavaScript
+ * whereas `NavigatorIOS` is a wrapper around `UINavigationController`.
  *
- * To change the animation or gesture properties of the scene, provide a
- * `configureScene` prop to get the config object for a given route. See
- * `Navigator.SceneConfigs` for default animations and more info on
- * scene config options.
+ * To set up the `Navigator` you provide one or more objects called routes,
+ * to identify each scene. You also provide a `renderScene` function that
+ * renders the scene for each route object.
+ *
+ * ```
+ * import React, { Component } from 'react';
```

Comment:
As a posterity note, if we were going to use the web player, this would be the example we would put it on. The others below are more snippets for illustration.